docs/new/common/authentication.md

```diff
@@ -62,7 +62,8 @@ To ensure that your authentication is set up correctly, you can run a test Subst
      map_valid_balance_changes \
      -e mainnet.eth.streamingfast.io:443 \
      --start-block 10397553 \
-     --stop-block +10
+     --stop-block +10 \
+     --netowork eth
```

Comment:
A better fix is actually to change the full invocation to:

```
substreams gui \
     erc20-balance-changes@latest \
     map_valid_balance_changes \
     --start-block 10397553 \
     --stop-block +10
```

This requires an up to date Substreams CLI.

Reply:
It works well!
b035cd45938ddd264c9664ad4300ca0be40dfd73